pkg/prompts/prompts.go

```diff
@@ -110,6 +111,11 @@ func validateTime(input string) error {
 	return err
 }
 
+func validateID(input string) error {
```

Comment:
I think we moved our validation functions to a different file.

Reply:
I thought so too, but that file doesn't exist! I guess some mishap during some merge....

Reply:
We'll extract this again in a follow up ticket: https://github.com/ava-labs/avalanche-cli/issues/477